### PR TITLE
Fix typos and sentences

### PR DESCRIPTION
**Pull request by MarieMinasyan**

### CODE REVIEW

File: cookbook/import_export/customize-import-behavior.rst [hunk 1]
```diff
@@ -1,3 +1,52 @@
 How to Customize the Import Behavior
 ====================================
```
Comment (nidup) [inline cookbook/import_export/customize-import-behavior.rst]: i dropped this page on purpose from the master, it's a deprecated way to do it, this page can be fix on 1.3 branch

Comment (MarieMinasyan) [inline cookbook/import_export/customize-import-behavior.rst]: Oups, rebase conflict problem, sorry.

Comment (nidup) [inline cookbook/import_export/customize-import-behavior.rst]: no pb, i plan to complete it in coming weeks :)

**Comment by nidup** [comment]
GTM

**Pull request merged**